test/jest/client/components/templates/district-office/district-office.test.js

```diff
@@ -431,4 +434,131 @@ describe('District Office template', () => {
     const documentLinks = getByTestId('office-document-links')
     expect(documentLinks).toBeInTheDocument()
   })
+  describe.only('Sucess Stories Component', () => {
```

Comment:
```suggestion
  describe('Success Stories Component', () => {
```
I think we can drop the only here